pages/dkp/kommander/2.2/DKP-insight/insight-items/index.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: layout.pug
+beta: false
+navigationTitle: Insight items
+title: Insight items
+excerpt: Insight items
+menuWeight: 40
+beta: true
+enterprise: false
+---
+
+The default insight items display in the DKP dashboard for the following types of information:
```

Comment:
```suggestion
The default Insights items display in the DKP dashboard for the following types of information:
```

Reply:
I think we could do a quick search and replaces for all instances where the product name is mentioned.

Reply:
Yes, i've still seen some locations. The folder all of these files are in should probably be "DKP Insights" as well.

Reply:
Let me confer with Product team on this. Product name is of course Insights, but when using the product we have "insight cards". Insights, because the customer UI view shows individual insight cards.

Reply:
Yes, thank you - I saw that in Kiran's notes above. So we'll go with "insight items in the Insights table", most likely.